src/site/content/en/blog/file-handling/index.md

```diff
@@ -163,7 +163,8 @@ added.
 
 ## Demo
 
-I have added file handling support to [Excalidraw][demo], a cartoon-style drawing app. When you
+I have added file handling support to [Excalidraw][demo], a cartoon-style drawing app. To test
+it, you first need to install Excalidraw. When you then
```

Comment:
Why would I think otherwise?

Reply:
Not you, but apparently previous readers.